src/pages/index.tsx

```diff
@@ -26,10 +26,7 @@ export default function Index({ data }: Props) {
     <Layout
       banner={
         <Banner href="/account/create">
-          <p>
-            Spokestack Maker - Self-Service Tools for Personal Voice Model
-            Creation
-          </p>
+          <p>Spokestack Maker - Painless voice interface prototyping!</p>
```

Comment:
```suggestion
          <p>Spokestack Maker - Painless Voice Interface Prototyping!</p>
```

We've mostly moved to titlecasing for headers and things.